### ISSUE TITLE
After sign-up check for possible duplicate Persons

### ISSUE THREAD

**Issue by smarziano** [smartchicago/kimball#179]
This relates to #78, but in terms of gift cards, we need to check this before we mail the gift card. As described in this comment https://github.com/smartchicago/kimball/issues/78#issuecomment-131968277, I currently check multiple pieces of information before deciding to send a gift card:
- Last name (sometimes duplicate last names are used because they are common names or because they are the same family) 
- Address (usually in excel I search by house number and direction and quickly go through the sheet to find duplicates)
- Email address
- Phone number

Here is an example of a search for duplicate: If a new sign-up shares a common last name but no other matching information (address, email, phone number), this person is unique. 

If a new sign-up shares the same last name plus other matching information (address, email, phone number), then if the other pieces of information are different (and there are not many repeating instances of this happening) then this person is unique. 

Example: John Smith signs up in January 2016 and Jane Smith signs up in February 2016 and both live at the same address but have different emails and phone numbers this is valid. 

Another example: If two sign-ups do not share the same last name but the same address, then other information like email and phone number need to be different for this to be a valid ID. 

**Task**
This is a very manual process and I like to check and email testers if something does not seem right. Therefore, I want to accept all new sign-ups but within Patterns designate if duplicate or not. 

At this add gift card stage I want a notification that a person might be a duplicate and who the duplicate might be.
<img width="1213" alt="screen shot 2016-07-25 at 10 31 24 am" src="https://cloud.githubusercontent.com/assets/5550969/17106978/f338ab1e-5252-11e6-9796-c48cdd05c926.png">

**Comment by smarziano** [comment]
What is currently on staging is a great start to checking for duplicate records especially with recent gift card sign-ups. I think that we will need to refine the address duplicate check because addresses can be the same but be entered differently (1234 West Belmont Ave vs. 1234 W Belmont are the same address).

I am going to add that this is Waiting on Merge, but keep future work in mind for this.

**Comment by smarziano** [comment]
This works well now, and as we go through our gift cards processes, I expect that we will come up with more ideas for improvements. Closing this issue and opened #204 related to the last comment in this thread.
